tests/test_kms/test_kms_boto3.py

```diff
@@ -1692,6 +1692,36 @@ def test_get_public_key():
     assert "EncryptionAlgorithms" not in public_key_response
 
 
+@mock_aws
+def test_get_public_key_with_alias():
+    client = boto3.client("kms", region_name="eu-west-1")
+    key = client.create_key(KeyUsage="SIGN_VERIFY", KeySpec="RSA_2048")
+    key_id = key["KeyMetadata"]["KeyId"]
+    client.create_alias(
+        AliasName="alias/foo",
+        TargetKeyId=key_id,
+    )
+    alias_arn = client.list_aliases()["Aliases"][0]["AliasArn"]
+
+    public_key_response_with_name = client.get_public_key(KeyId="alias/foo")
+    public_key_response_with_arn = client.get_public_key(KeyId=alias_arn)
+
+    assert "PublicKey" in public_key_response_with_name, (
+        "PublicKey" in public_key_response_with_arn
+    )
```

Comment:
The second argument to `assert` is a message to inform the user what went wrong - you can't use it to chain assertions unfortunately.

As an example, this passes:
```
assert True, False
```
Because the `False` statement would only be used as a message if the condition (first argument) would be False.

But this example fails, and gives us the second argument as a description as to why it fails:
```
>>> assert False, True
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
AssertionError: True
```

TLDR: These assertions should probably just be split up:
```suggestion
    assert "PublicKey" in public_key_response_with_name
    assert "PublicKey" in public_key_response_with_arn
```

Reply:
Thanks @bblommers for the explanation, I'm not a true python dev _yet_ so it's helpful 👍 

The requested change has been pushed !